iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/PropertiesXml.java

```diff
@@ -24,18 +25,30 @@
     public static void write(Identifiable<?> identifiable, NetworkXmlWriterContext context) throws XMLStreamException {
         if (identifiable.hasProperty()) {
             for (String name : identifiable.getPropertyNames()) {
-                String value = identifiable.getProperty(name);
-                context.getWriter().writeEmptyElement(context.getVersion().getNamespaceURI(), PROPERTY);
-                context.getWriter().writeAttribute(NAME, name);
-                context.getWriter().writeAttribute(VALUE, value);
+                if (!name.equals(Substation.GEOGRAPHICAL_TAGS_KEY)) {
+                    String value = identifiable.getProperty(name);
+                    context.getWriter().writeEmptyElement(context.getVersion().getNamespaceURI(), PROPERTY);
+                    context.getWriter().writeAttribute(NAME, name);
+                    context.getWriter().writeAttribute(VALUE, value);
+                } else {
+                    String value = context.getAnonymizer().anonymizeString(identifiable.getProperty(name));
+                    context.getWriter().writeEmptyElement(context.getVersion().getNamespaceURI(), PROPERTY);
+                    context.getWriter().writeAttribute(NAME, name);
+                    context.getWriter().writeAttribute(VALUE, value);
+                }
```

Comment:
I'm really not sure you should keep the anonymization for geographicalTags... what do you think @mathbagu ? For me, it adds complexity for a feature which is not very used...